common/v2/services/Store/DataManager/DataProvider.tsx

```diff
@@ -53,17 +53,20 @@ export const DataProvider: React.FC = ({ children }) => {
   /*
    * Manage sync between appState an db
    */
-  const syncDb = useCallback(() => {
-    console.debug('Updating LocalStorage');
-    updateDb(deMarshallState(appState));
-  }, [appState]);
+  const syncDb = useCallback(
+    state => {
+      console.debug('Updating LocalStorage');
+      updateDb(deMarshallState(state));
+    },
+    [appState]
+  );
   // By default we sync no more than once a minute
   useThrottleFn(syncDb, 60000, [appState]);
```

Comment:
As discussed let's reduce the the throttle to 30000

Reply:
Done